app/src/components/domain/RiskImminentEvents/Pdc/EventDetails/index.tsx

```diff
@@ -18,6 +45,9 @@ interface Props {
     data: PdcEventItem;
     exposure: PdcExposure | undefined;
     pending: boolean;
+    onLayerChange: (value: boolean, name: LayerType) => void;
+    layers: Record<LayerType, boolean>;
```

Comment:
We should also rename this prop. Isn't this layer visibility?

Reply:
This a mapped object of all active layers.

Reply:
let's rename the prop names accordingly